src/mpi/coll/algorithms/treealgo/treealgo.c

```diff
@@ -62,14 +65,162 @@ int MPIR_Treealgo_tree_create(int rank, int nranks, int tree_type, int k, int ro
     MPIR_FUNC_EXIT;
 
   fn_exit:
+    if (MPIR_CVAR_TREE_DUMP) {
```

Comment:
"TREE_DUMP" is too ambiguous for one to guess what it does. How about `MPIR_CVAR_COLL_TREE_DEBUG`?

Reply:
Changed to "MPIR_CVAR_COLL_TREE_DUMP".